src/app/cases/viewer/case-viewer.component.ts

```diff
@@ -43,13 +44,13 @@ export class CaseViewerComponent implements OnInit, OnDestroy {
     // Clone and sort tabs array
     this.sortedTabs = this.orderService.sort(this.caseDetails.tabs);
 
-    // Clone and sort fields array
-    this.sortedTabs = this.sortedTabs.map(tab => Object.assign({}, tab, {
-      fields: this.orderService.sort(tab.fields)
-    }));
-
     this.caseFields = this.getTabFields();
 
+    // Clone and sort fields array
+    this.sortedTabs = this.sortedTabs
```

Comment:
Can we extract the right side of assignment to a well named method? Maybe the comment could be removed as well.

Reply:
👍